apps/testclient/templates/testlinks.html

```diff
@@ -30,9 +30,13 @@
 
 <body class="ds-base">
  <h1>API Test Page</h1>
- 
+
+ <p><b>IMPORTANT:</b>  {% if user.is_authenticated %}It looks like you may be logged in to your Sandbox Developer Account. {% endif %}
+       Being logged into your Developer Account when using this Test Client triggers a 500 error.</p>
+ <p><a href="{% url 'logout' %}">Logout</a> now.</p>
```

Comment:
Should the section including the Logout be enclosed in the {% endif %}  ?

Reply:
I think it is safe to provide a logout but for cleanliness we could include the whole paragraph in the if..endif section.

Reply:
If we do this we need to move the IMPORTANT: inside too.